boards/m5stack/m5stack_cores3/m5stack_cores3-pinctrl.dtsi

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2024 Zhang Xingtao <zhxt@live.cn>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/dt-bindings/pinctrl/esp-pinctrl-common.h>
+#include <dt-bindings/pinctrl/esp32s3-pinctrl.h>
+#include <zephyr/dt-bindings/pinctrl/esp32s3-gpio-sigmap.h>
+
+
+&pinctrl {
+	uart0_default: uart0_default {
+		group1 {
+			pinmux = <UART0_TX_GPIO43>;
+			output-high;
+		};
+		group2 {
+			pinmux = <UART0_RX_GPIO44>;
+			bias-pull-up;
+		};
+	};
+
+	spim2_default: spim2_default {
+		group1 {
+			pinmux = <SPIM2_MISO_GPIO8>,
```

Comment:
The SPI uses 35-37 pins.

Please consider applying https://github.com/zephyrproject-rtos/zephyr/pull/79765/commits/30e4e0c9f6b2ab9bde11aa1fa8c1a9f8628cf75e

Reply:
> The SPI uses 35-37 pins.
> 
> Please consider applying [30e4e0c](https://github.com/zephyrproject-rtos/zephyr/commit/30e4e0c9f6b2ab9bde11aa1fa8c1a9f8628cf75e)

Yeah, I had fixed that last time, but haven't  submit yet.